var/spack/repos/builtin/packages/alps/package.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class Alps(CMakePackage):
+    """Algorithms for Physics Simulations
+
+    Tags: Condensed Matter Physics, Computational Physics
+    """
+
+    homepage = "https://github.com/ALPSim/ALPS"
+    url = "https://github.com/ALPSim/ALPS/archive/refs/tags/v2.3.3-beta.5.tar.gz"
+
+    maintainers("Sinan81")
```

Comment:
@iskakof I'm wondering if you might be interested in being listed as maintainer here so that users can contact you for build issues.

Reply:
can be done later. resolving.